1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -17,7 +17,14 @@ const BIRTHDAYS = [
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+    let i = 0;
+
+  while (i < BIRTHDAYS.length) {
```

Comment:
Instead of using `BIRTHDAYS` here, you should be using the argument being passed into the function - `birthdays`.